CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 - Improved error message when invalid network backend provided
+- Upgraded Rust toolchain from 1.64.0 to 1.66.0.
```

Comment:
Just double-checking that 1.64.0 is correct here. What was the latest toolchain that we released with?

Reply:
I upgraded from 1.65 to 1.66 but the PR for the upgrade from 1.64 to 1.65 didn't contain any CHANGELOG modifications so I wanted to fix that as well. I didn't want to have 2 separate lines for 1.64 -> 1.65 and 1.65 -> 1.66 as long as looked redundant to me. Should I change this into 2 separate changes, however ?

Reply:
From the release commit for 1.2.0 we can see its 1.64.0.
https://github.com/firecracker-microvm/firecracker/blob/408f2fca1a3a55027245513531bd6abca8e39017/tools/devctr/Dockerfile.x86_64#L7

Reply:
Yes, and on 5.12.2022 (which was after the 1.2 release that took place on 30.11.2022: https://github.com/firecracker-microvm/firecracker/commit/408f2fca1a3a55027245513531bd6abca8e39017), a PR for the upgrade to 1.65 has been merged: https://github.com/firecracker-microvm/firecracker/commit/4ae37a4d0896d73a8a684550e22f4440c10b94a4

Reply:
That means the upgrade to 1.65 will be part of FC 1.3 and the changelog didn't contain anything related to rust upgrade 1.64->1.65 in the `Unreleased` section.